V6_MIGRATION.md

```diff
@@ -52,12 +52,16 @@ The possible values for `BTVenmoPaymentMethodUsage` include:
 * `.multiUse` - the Venmo payment will be authorized for future payments and can be vaulted.
 * `.singleUse` - the Venmo payment will be authorized for a one-time payment and cannot be vaulted.
 
+`BTVenmoClient.tokenizeVenmoAccount(with: BTVenmoRequest, completion: @escaping (BTVenmoAccountNonce?, Error?) -> Void)` has been renamed to `BTVenmoClient.tokenize(_ request: BTVenmoRequest, completion: @escaping (BTVenmoAccountNonce?, Error?) -> Void)`
+
```

Comment:
Nitpick - do we want to use the shorthand syntax (like we looked at at the top of Apple's dev docs) here instead?

I think it would be:
```suggestion
`BTVenmoClient.tokenizeVenmoAccount(with:completion:)` has been renamed to `BTVenmoClient.tokenize(_:completion:)`

```

Reply:
It looks like we've followed the pattern in this PR at least for the rest of the V6_MIGRATION guide. I'm going to 🏈  for now and add a ticket for us to decide and update everything in here to match the decided pattern. Ticket here: DTBTSDK-2518. Also included an audit of the CHANGELOG in the ticket since it's probably be good to make sure we are consistent there for v6 as well.